src/selectors.ts

```diff
@@ -52,13 +52,17 @@ export const selectProjectsWithTransferrableSites = createSelector(
           };
         }),
     );
-    let projectRecord = projects.reduce(
+
+    const unaffiliatedSites = Object.values(sites)
+      .filter(({ projectId }) => projectId === undefined)
```

Comment:
question: do you need to check for undefined specifically? Or any false value?

Reply:
The value is either a `string` or `undefined`. Just the boolean value would silently discard an empty string. We don't expect an empty string for the project ID, but I'd rather be explicit in the case that it makes the error slightly easier to track down.